### ISSUE TITLE
Issue in landscape exported video.

### ISSUE THREAD

**Issue by aleberguer** [rs/SDAVAssetExportSession#57]
Hi, I'm having an issue when I trying to export a video in landscape mode.
This is the original video
![screen shot 2016-06-09 at 11 15 51 am](https://cloud.githubusercontent.com/assets/5271876/15933170/100f77c8-2e35-11e6-9e52-3b00cfdad9e4.png)
And the exported video is cut in half.
![screen shot 2016-06-09 at 11 16 56 am](https://cloud.githubusercontent.com/assets/5271876/15933169/100afb30-2e35-11e6-8486-30895fa63c86.png)

my configuration for the encoder is this 
`let encoder = SDAVAssetExportSession(asset: sourceAsset)
    encoder.outputFileType = AVFileTypeMPEG4;
    encoder.outputURL = savePathUrl;

```
encoder.videoSettings = [
    AVVideoCodecKey: AVVideoCodecH264,
    AVVideoWidthKey: 900,
    AVVideoHeightKey: 506,
    AVVideoCompressionPropertiesKey: [

        AVVideoAverageBitRateKey: 725000,
        AVVideoProfileLevelKey: AVVideoProfileLevelH264High40,
    ]
]

encoder.audioSettings = [
    AVFormatIDKey: NSNumber(int: Int32(kAudioFormatMPEG4AAC)),
    AVNumberOfChannelsKey: 2,
    AVSampleRateKey: 44100,
    AVEncoderBitRateKey: 64000
]
encoder.shouldOptimizeForNetworkUse = true
```

`
the original video is 1280x720.

Any suggestions? Thanks.

**Comment by torrao** [comment]
@aleberguer I'm also struggling with the same issue. Did you solved it?

**Comment by aleberguer** [comment]
I realized that the output frame is always in portrait, isn't checking the video orientation. The visible part of the video is equal to the width in portrait.
My application only records videos in landscape mode. I made a workaround and  could make it work doing a translation above the line 333

`transform = CGAffineTransformConcat(transform,  CGAffineTransformMakeTranslation(naturalSize.width - naturalSize.height, 0));`

I'm sure there is a better solution for this, but I had no time and for my case this works perfectly.
